README.md

```diff
@@ -49,13 +73,13 @@ class { 'wsus_client':
 
 ## Usage
 
+### Schedule automatic updates
 
+To schedule automatic updates from a WSUS server, declare the `wsus_client` class with a WSUS [`server_url`][] as well as the [`auto_update_option`][], [`scheduled_install_day`][], and [`scheduled_install_hour`][] parameters.
```

Comment:
'from a wsus server' makes me think I need to run this on my WSUS server
'To schedule updates to be pulled from a given wsus server' Also 'schedule automatic' seems a bit redundant